spec/dnsimple/client/services_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'spec_helper'
+
+describe Dnsimple::Client, ".services" do
+
+  subject { described_class.new(base_url: "https://api.dnsimple.test", access_token: "a1b2c3").services }
```

Comment:
👍 I think we can keep track of all the endpoints that should work without authentication on #74.